### DISCUSSION TITLE
Can't say pirating.

### DISCUSSION THREAD

**Discussion by maboroshin** [KRTirtho/spotube#533] [Q&A]
I think pirating is a violation of rights. YouTube removes videos that violate copyright. This deletion does not seem to be working perfectly. We will often be watching videos that have no copyright issues. In this case, the copying is permitted as "personal use" (or fair use).

RIAA (Recording Industry Association of America) also states in https://www.riaa.com/resources-learning/about-piracy/.

> It’s okay to download and stream music from sites authorized by the owners of the copyrighted music, whether or not such sites charge a fee. Visit [www.whymusicmatters.com](http://www.whymusicmatters.com/) for a list of a number of authorized and safe sites where permission is granted and content is available for downloading and/or streaming.

> It’s never okay to download unauthorized music from **pirate** sites (web or FTP) or peer-to-peer systems, such as BitTorrent.

Downloading from YouTube would be legally permissible for personal use. Uploading this data is piracy. Pirate site is uploaded as piracy. YouTube is not a pirate site.

**Comment by Lucyy-cloud** [comment]
If you want to obtain the license of Amazon Music and the content can be downloaded and/or streamed music list, you can use the[ Viwizard Amazon Music Converter](https://www.viwizard.com/amazon-music-converter/) to download songs offline and convert them into MP3, FLAC, WAV and other files for personal use.

**Comment by ovizii** [comment]
What is your question here?
